test/format/format_config.c

```diff
@@ -1516,40 +1528,44 @@ config_disagg_storage(void)
     page_log = GVS(DISAGG_PAGE_LOG);
 
     g.disagg_storage_config = (strcmp(page_log, "off") != 0 && strcmp(page_log, "none") != 0);
-    if (g.disagg_storage_config) {
-        if (!config_explicit(NULL, "disagg.mode")) {
-            /* Randomly assign "leader" or "follower" to disagg.mode with equal probability. */
-            testutil_snprintf(buf, sizeof(buf), "disagg.mode=%s",
-              mmrand(&g.data_rnd, 1, 100) <= 50 ? "leader" : "follower");
-            config_single(NULL, buf, false);
-        }
+    if (!g.disagg_storage_config)
+        return; /* Disaggregated storage not enabled. */
 
-        mode = GVS(DISAGG_MODE);
-        if (strcmp(mode, "leader") != 0 && strcmp(mode, "follower") != 0 &&
-          strcmp(mode, "switch") != 0)
-            testutil_die(EINVAL, "illegal disagg.mode configuration: %s", mode);
+    if (!config_explicit(NULL, "disagg.mode")) {
+        /* Randomly assign "leader" or "follower" to disagg.mode with equal probability. */
+        testutil_snprintf(buf, sizeof(buf), "disagg.mode=%s",
+          mmrand(&g.data_rnd, 1, 100) <= 50 ? "leader" : "follower");
+        config_single(NULL, buf, false);
+    }
 
-        if (strcmp(mode, "switch") == 0)
-            /* Randomly assign "leader" or "follower". */
-            g.disagg_leader = mmrand(&g.data_rnd, 0, 1);
-        else
-            g.disagg_leader = strcmp(mode, "leader") == 0;
+    mode = GVS(DISAGG_MODE);
+    if (strcmp(mode, "leader") != 0 && strcmp(mode, "follower") != 0 && strcmp(mode, "switch") != 0)
+        testutil_die(EINVAL, "illegal disagg.mode configuration: %s", mode);
 
-        /* Disaggregated storage requires timestamps. */
-        config_off(NULL, "transaction.implicit");
-        config_single(NULL, "transaction.timestamps=on", true);
+    if (strcmp(mode, "switch") == 0)
+        /* Randomly assign "leader" or "follower". */
+        g.disagg_leader = mmrand(&g.data_rnd, 0, 1);
+    else
+        g.disagg_leader = strcmp(mode, "leader") == 0;
 
-        /* It makes sense to do checkpoints. */
-        if (!config_explicit(NULL, "checkpoint"))
-            config_single(NULL, "checkpoint=on", false);
+    /* Configure page deltas. */
+    config_disagg_delta("disagg.internal_page_delta", GV(DISAGG_INTERNAL_PAGE_DELTA));
```

Comment:
I dont think we need these functions anymore? 🤔

Reply:
Removed.